### ISSUE TITLE
pdbinpath vs PDBINDIR on w32 builds

### ISSUE THREAD

**Issue by umlaeute** [pure-data/pd-lib-builder#34]
it seems that (with 872c34b1d3851937251b45a28ba8a91f34c2e2a6) the `PDBINDIR` var is not honoured. instead the `pdbinpath` is used. shouldn't it be the other way around if `PDDIR` is set?

~~~
$ export CC=gcc
$ mingw32-make PDDIR="Pd"
gcc -DPD -I "C:\projects\libdir./Pd/src" -DMSW -DNT   -DVERSION='"1.11"' -Wall -Wextra -Wshadow -Winline -Wstrict-aliasing -O3 -ffast-math -funroll-loops -fomit-frame-pointer -march=pentium4 -msse -msse2 -mfpmath=sse -o foo.o -c foo.c
++++ info: linking objects in foo.dll for lib foo
gcc -static-libgcc -shared -Wl,--enable-auto-import "/pd.dll"    -o foo.dll foo.o
gcc.exe: error: C:/Program Files/Git/pd.dll: No such file or directory
Makefile.pdlibbuilder:796: recipe for target 'foo.dll' failed
mingw32-make: *** [foo.dll] Error 1
~~~

**Comment by umlaeute** [comment]
fwiw, calling it with `pdbinpath` set, makes the build succeed:

~~~
$ mingw32-make PDDIR="Pd" pdbinpath="Pd/bin"
ls: cannot access '/pd/src': No such file or directory
++++ info: using Makefile.pdlibbuilder version 0.4.4
++++ info: using Pd API Pd/src/m_pd.h
++++ info: making target all in lib foo
++++ info: making foo.o in lib foo
gcc -DPD -I "Pd/src" -DMSW -DNT   -DVERSION='"1.11"' -Wall -Wextra -Wshadow -Winline -Wstrict-aliasing -O3 -ffast-math -funroll-loops -fomit-frame-pointer -march=pentium4 -msse -msse2 -mfpmath=sse -o foo.o -c foo.c
++++ info: linking objects in foo.dll for lib foo
gcc -static-libgcc -shared -Wl,--enable-auto-import "Pd/bin/pd.dll"    -o foo.dll foo.o     
++++info: target all in lib foo completed
$
~~~

**Comment by katjav** [comment]
Fixed with commit 7740128.
